projects/tuist/fixtures/app_with_build_rules/Project.swift

```diff
@@ -0,0 +1,26 @@
+import ProjectDescription
+
+let project = Project(
```

Comment:
Please fix the indentation using 4 spaces 🙏

Reply:
fixed